### PR TITLE
Added missing Debian 11 info

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Some Debian 11 related info was missing from the various pages.
This PR fixes that.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3
- [x] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by keichwa — APPROVED** [review]
LGTM!

File: modules/client-configuration/pages/supported-features.adoc [hunk 1]
```diff
@@ -107,8 +107,13 @@ ifeval::[{suma-content} == true]
...
 | {check}
 
+| {debian} 11
+| {x86_64}, {aarch64}, armv7l, i586
+| {cross}
+| {check}
+
 | {debian} 10
-| {x86_64}
+| {x86_64}, {aarch64}, armv7l, i586
```
[the remark this answers is not in the document]
Comment (juliogonzalez) [reply modules/client-configuration/pages/supported-features.adoc:116]: SUSE Manager: https://scc.suse.com/admin/products/2410
Uyuni: https://www.uyuni-project.org/doc/2022.08/release-notes-uyuni-server.html#_debian_11_as_client

[the remark this answers is not in the document]
Comment (0rnela) [reply modules/client-configuration/pages/supported-features.adoc]: Thanks, implemented.

**Review by jcayouette — APPROVED** [review]

File: modules/administration/pages/ansible-compliance-as-code.adoc [hunk 2]
```diff
@@ -268,4 +268,9 @@ debian10-script-anssi_np_nt28_high.sh +
 debian10-script-anssi_np_nt28_minimal.sh +
 debian10-script-anssi_np_nt28_restrictive.sh +
 debian10-script-standard.sh +
+debian11-script-anssi_np_nt28_average.sh +
```
[the remark this answers is not in the document]
Comment (meaksh) [reply modules/administration/pages/ansible-compliance-as-code.adoc:271]: Hey, yes, it seems those are correct:

```console
root@923d58b43f43:/# dpkg -L scap-security-guide-debian | grep sh$
/usr/share/scap-security-guide/bash
/usr/share/scap-security-guide/bash/debian10-script-anssi_np_nt28_average.sh
/usr/share/scap-security-guide/bash/debian10-script-anssi_np_nt28_high.sh
/usr/share/scap-security-guide/bash/debian10-script-anssi_np_nt28_minimal.sh
/usr/share/scap-security-guide/bash/debian10-script-anssi_np_nt28_restrictive.sh
/usr/share/scap-security-guide/bash/debian10-script-standard.sh
/usr/share/scap-security-guide/bash/debian11-script-anssi_np_nt28_average.sh
/usr/share/scap-security-guide/bash/debian11-script-anssi_np_nt28_high.sh
/usr/share/scap-security-guide/bash/debian11-script-anssi_np_nt28_minimal.sh
/usr/share/scap-security-guide/bash/debian11-script-anssi_np_nt28_restrictive.sh
/usr/share/scap-security-guide/bash/debian11-script-standard.sh
/usr/share/scap-security-guide/bash/debian9-script-anssi_np_nt28_average.sh
/usr/share/scap-security-guide/bash/debian9-script-anssi_np_nt28_high.sh
/usr/share/scap-security-guide/bash/debian9-script-anssi_np_nt28_minimal.sh
/usr/share/scap-security-guide/bash/debian9-script-anssi_np_nt28_restrictive.sh
/usr/share/scap-security-guide/bash/debian9-script-standard.sh
```

But, I just notice that the above path "Bash Script Folder" in this document is not correct, it is missing the `bash` subfolder apparentely.

[the remark this answers is not in the document]
Comment (0rnela) [reply modules/administration/pages/ansible-compliance-as-code.adoc:271]: Good catch, thanks @meaksh.

**Pull request merged**